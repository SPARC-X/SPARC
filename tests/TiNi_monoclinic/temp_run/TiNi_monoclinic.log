
fchrg = 2.50000000 > 0.0 (icmod != 0)
This pseudopotential contains non-linear core correction. 

fchrg = 2.500000, READING MODEL CORE CHARGE!


fchrg = 3.00000000 > 0.0 (icmod != 0)
This pseudopotential contains non-linear core correction. 

fchrg = 3.000000, READING MODEL CORE CHARGE!

The program took 50.141 s.
