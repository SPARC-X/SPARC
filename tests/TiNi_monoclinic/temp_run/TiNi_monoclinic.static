Total free energy (Ha): -4.543699721595847E+02
Atomic forces (Ha/Bohr):
  4.7778476133E-02  -2.1075870823E-06   6.2971274283E-03
 -4.7778473509E-02  -2.5324453071E-06  -6.2969544565E-03
 -3.7201450726E-02   5.7984101804E-06  -1.0957955814E-01
  3.7201448102E-02  -1.1583777910E-06   1.0957938517E-01
Stress (GPa): 
  3.1477562213E+01   3.9606918500E-05  -8.3292816899E+00 
  3.9606918500E-05   4.0643588256E+00   3.7681622841E-05 
 -8.3292816899E+00   3.7681622841E-05  -5.0285826463E+01
