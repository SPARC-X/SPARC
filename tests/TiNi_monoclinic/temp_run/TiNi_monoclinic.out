***************************************************************************
*                       SPARC (version Apr 23, 2021)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Mon May 10 21:07:09 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 7.66285515 5.50856295 8.98377642 
LATVEC:
0.981853167430541 0 -0.189642710396718 
0 1 0 
0 0 1 
FD_GRID: 77 55 90
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001000000336
CHEB_DEGREE: 51
NSTATES: 41
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: TiNi_monoclinic
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 6
NP_DOMAIN_PARAL: 2 1 2
NP_DOMAIN_PHI_PARAL: 3 2 4
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  24
Mesh spacing in x-direction        :  0.0995176 (Bohr)
Mesh spacing in y-direction        :  0.100156 (Bohr)
Mesh spacing in z direction        :  0.0998197 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  TiNi_monoclinic.out
Total number of atom types         :  2
Total number of atoms              :  4
Total number of electrons          :  60
Atom type 1  (valence electrons)   :  Ti 12
Pseudopotential                    :  Ti.psp8
Pseudocharge radii of atom type 1  :  6.37 6.41 6.39 (x, y, z dir)
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  Ni 18
Pseudopotential                    :  Ni.psp8
Pseudocharge radii of atom type 2  :  6.27 6.21 6.29 (x, y, z dir)
Number of atoms of type 2          :  2
Estimated total memory usage       :  796.78 MB
Estimated memory per processor     :  33.20 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -1.1347069511E+02        6.559E-02        7.781
2            -1.1347549788E+02        1.675E-01        2.287
3            -1.1356169948E+02        8.254E-02        2.252
4            -1.1353717626E+02        1.215E-01        2.242
5            -1.1354863601E+02        1.104E-01        2.265
6            -1.1357979842E+02        5.265E-02        2.267
7            -1.1358741900E+02        3.727E-02        2.250
8            -1.1359131014E+02        1.625E-02        2.245
9            -1.1359226943E+02        7.037E-03        2.212
10           -1.1359187771E+02        1.221E-02        2.204
11           -1.1359199605E+02        1.175E-02        2.209
12           -1.1359230850E+02        6.878E-03        2.200
13           -1.1359248768E+02        1.211E-03        2.178
14           -1.1359249222E+02        4.775E-04        2.151
15           -1.1359249289E+02        2.182E-04        2.081
16           -1.1359249305E+02        4.676E-05        2.094
17           -1.1359249306E+02        8.737E-06        2.034
18           -1.1359249304E+02        7.055E-06        2.007
19           -1.1359249306E+02        1.486E-06        1.992
20           -1.1359249304E+02        3.507E-07        1.918
Total number of SCF: 20    

k = [0.000, 0.000, 0.000]
Occupation of state 37 = 0.000000000000000.
Occupation of state 41 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -1.1359249304E+02 (Ha/atom)
Total free energy                  : -4.5436997216E+02 (Ha)
Band structure energy              : -4.4631946303E+01 (Ha)
Exchange correlation energy        : -6.0416035577E+01 (Ha)
Self and correction energy         : -6.0243655015E+02 (Ha)
Entropy*kb*T                       : -9.1210890155E-04 (Ha)
Fermi level                        :  4.0640461748E-01 (Ha)
RMS force                          :  8.1956885048E-02 (Ha/Bohr)
Maximum force                      :  1.1572219982E-01 (Ha/Bohr)
Time for force calculation         :  0.147 (sec)
Pressure                           :  4.9146351418E+00 (GPa)
Maximum stress                     :  5.0285826463E+01 (GPa)
Time for stress calculation        :  0.268 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  50.140 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
